README.md

```diff
@@ -10,7 +10,7 @@ This package will work in any PHP application, and Facades and Service Providers
 
 Run the following Composer command in your terminal, or simply add `'ryannielson/meta': '1.1.*'` to your composer.json file:
 
-    composer require ryannielson/meta:'1.1.*'
+    composer require austerussian/meta
```

Comment:
Excuse me. I learned to make a "Fork" on GithuB and misunderstood what I need to do. I just need a package with my changes. Excuse for troubling.